docs/api/cli-options.md

```diff
@@ -76,6 +76,143 @@ Options include:
+
+### `upgrade`
+
+Upgrade your Storybook instance to the latest version. Read more in the [upgrade guide](../configure/upgrading.md).
+
+```shell
+storybook upgrade [options]
+```
+
+Options include:
+
+| Option               | Description                                                                                                                                                                 |
+| -------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `-c`, `--config-dir` | Directory where to load Storybook configurations from<br/>`storybook upgrade --config-dir .storybook`                                                                       |
+| `-n`, `--dry-run`    | Checks for version upgrades without installing them<br/>`storybook upgrade --dry-run`                                                                                       |
+| `-s`, `--skip-check` | Skips the migration check step during the upgrade process<br/> `storybook upgrade --skip-check`                                                                             |
+| `-y`, `--yes`        | Skips interactive prompts and automatically upgrades Storybook to the latest version<br/>`storybook upgrade --yes`                                                          |
+| `--package-manager`  | Sets the package manager to use when installing the addon.<br/> Available package managers include `npm`, `yarn`, and `pnpm`<br/>`storybook upgrade --package-manager pnpm` |
+
+### `info`
+
+Reports useful debugging information about your environment. Helpful in providing information when opening an issue or a discussion.
+
+```shell
+storybook info
+```
+
+Example output:
+
+```shell
+Storybook Environment Info:
+
+  System:
+    OS: macOS 14.2
+    CPU: (8) arm64 Apple M3
+    Shell: 5.9 - /bin/zsh
+  Binaries:
+    Node: 18.19.0 - ~/.nvm/versions/node/v18.19.0/bin/node
+    Yarn: 1.22.21 - /usr/local/bin/yarn <----- active
+    npm: 10.2.3 - ~/.nvm/versions/node/v18.19.0/bin/npm
+    pnpm: 8.12.0 - /opt/homebrew/bin/pnpm
+  Browsers:
+    Chrome: 120.0.6099.199
+    Safari: 17.2
+  npmPackages:
+    @storybook/addon-a11y: ^7.6.6 => 7.6.6
+    @storybook/addon-essentials: ^7.6.6 => 7.6.6
+    @storybook/addon-interactions: ^7.6.6 => 7.6.6
+    @storybook/addon-links: ^7.6.6 => 7.6.6
+    @storybook/addon-onboarding: ^1.0.10 => 1.0.10
+    @storybook/blocks: ^7.6.6 => 7.6.6
+    @storybook/preset-create-react-app: ^7.6.6 => 7.6.6
+    @storybook/react: ^7.6.6 => 7.6.6
+    @storybook/react-webpack5: ^7.6.6 => 7.6.6
+    @storybook/server-webpack5: ^7.6.6 => 7.6.6
+    @storybook/test: ^7.6.6 => 7.6.6
+    @storybook/test-runner: ^0.16.0 => 0.16.0
+    chromatic: ^10.2.0 => 10.2.0
+    msw-storybook-addon: ^1.10.0 => 1.10.0
+    storybook: ^7.6.6 => 7.6.6
+```
```

Comment:
Maybe we should hide this in a `details`, just to help the page not be so long?

````suggestion
<details>
<summary>Example output</summary>

```shell
Storybook Environment Info:

  System:
    OS: macOS 14.2
    CPU: (8) arm64 Apple M3
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 18.19.0 - ~/.nvm/versions/node/v18.19.0/bin/node
    Yarn: 1.22.21 - /usr/local/bin/yarn <----- active
    npm: 10.2.3 - ~/.nvm/versions/node/v18.19.0/bin/npm
    pnpm: 8.12.0 - /opt/homebrew/bin/pnpm
  Browsers:
    Chrome: 120.0.6099.199
    Safari: 17.2
  npmPackages:
    @storybook/addon-a11y: ^7.6.6 => 7.6.6
    @storybook/addon-essentials: ^7.6.6 => 7.6.6
    @storybook/addon-interactions: ^7.6.6 => 7.6.6
    @storybook/addon-links: ^7.6.6 => 7.6.6
    @storybook/addon-onboarding: ^1.0.10 => 1.0.10
    @storybook/blocks: ^7.6.6 => 7.6.6
    @storybook/preset-create-react-app: ^7.6.6 => 7.6.6
    @storybook/react: ^7.6.6 => 7.6.6
    @storybook/react-webpack5: ^7.6.6 => 7.6.6
    @storybook/server-webpack5: ^7.6.6 => 7.6.6
    @storybook/test: ^7.6.6 => 7.6.6
    @storybook/test-runner: ^0.16.0 => 0.16.0
    chromatic: ^10.2.0 => 10.2.0
    msw-storybook-addon: ^1.10.0 => 1.10.0
    storybook: ^7.6.6 => 7.6.6
```

</details>
````

Reply:
We can remove some of the addons to match the ones provided by default and the same goes for browsers and package managers, adding only one, condensing the information without the need for a collapsible item here.